apps/fabric-website/package.json

```diff
@@ -51,6 +51,7 @@
     "color-functions": "1.1.0",
     "json-loader": "^0.5.7",
     "office-ui-fabric-react": ">=6.106.2 <7.0.0",
-    "tslib": "^1.7.1"
+    "tslib": "^1.7.1",
+    "whatwg-fetch": "2.0.4"
```

Comment:
This package had an implicit dependency on whatwg-fetch (one of the files in the package imports it). With pnpm, such dependencies must be made explicit.